content/docs/rfcs/14/README.md

```diff
@@ -82,11 +94,17 @@ message WakuMessage {
   optional bool ephemeral = 31;
 }
 ```
+## Waku Message Vaildator
```

Comment:
This specification does not explicitly mention message validation. As detailed in [libp2p spec](https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/gossipsub-v1.1.md), there is a message validator. Also in Waku node implements a validator here, [message_validator.ts](https://github.com/waku-org/js-waku/blob/master/packages/relay/src/message_validator.ts).